jobs/pipeline-jobs.yml

```diff
@@ -1,47 +1,49 @@
 
-- job:
-    name: familylife-forms-api
+- project:
+    name: FamilyLife-FormsAPI
```

Comment:
@vdymna-fl 
Do you mind if I change the two FL job names to match their github repo names? See the commit message here for the reason.

familylife-forms-api -> FamilyLife-FormsAPI
familylife-aop-api -> FamilyLife-ArtOfParentingAPI

Reply:
@mattdrees yes, you can rename them for sure.